src/main/java/io/nats/client/Options.java

```diff
@@ -1391,8 +1464,16 @@ else if (OPENTLS_PROTOCOL.equals(serverURI.getScheme())) {
         }
     }
 
+    // ----------------------------------------------------------------------------------------------------
+    // CONSTRUCTOR
+    // ----------------------------------------------------------------------------------------------------
     private Options(Builder b) {
-        this.servers = b.servers;
+        this.servers = new ArrayList<>(b.servers); // builder servers is a set so no dupes
+        refinedServers = new ArrayList<>(this.servers.size()); // refined just turns uris into strings since that's what's really used

```

Comment:
is refined de-duped?

Reply:
it was not de-duped, all it was was a String version of the URI list. IO removed it anyway as part of simplification